design/CERTIFICATE_SIGNING.md

```diff
@@ -1,32 +1,100 @@
 # Certificate Signing - Overview
 
-Certificate authority (CA) signing in Conjur adds support for signing
-host certificates using a CA certificate and private key stored in
-Conjur.
+Version: Alpha
 
-A primary use case this supports is configuring mutual TLS between
-hosts with machine identity provisioned in Conjur. In this scenario,
-Conjur operators can configure a signing CA in conjur with a key and
-issuer certificate created outside of Conjur. Hosts may be granted
-the privilege to sign their own short-lived certificate through the CA.
-This host certificate can then be used to identify the service to
-other hosts or systems that trust the issuing CA.
+Implementation Owner:
+@micahlee
 
-## Getting Started
+## Motivation
 
-To begin signing host certificates you need:
+Digital certificates are a useful mechanism for authentication, data
+integrity, and data confidentiality. Certificates offer many advantages
+over other options such as passwords, API keys, or public/private key
+pairs because they are revokable, have built-in mechanisms for validity
+checking, may include attributes describing the subject they identify,
+and most importantly, they can be signed by a trusted authority.
 
-- A running Conjur instance
-- A policy to configure the CA and hosts to request certificates
-- A certificate chain and private key for the issuer CA
+The use of trusted authorities, and a public key infrastructure (PKI)
+make digital certificates (and certificate signing) a cornerstone of
+flexible and scalable access control architecture.
 
-### Policy for Conjur CAs
+Managing private key material for trusted authorities, and controlling
+and auditing the use of signatures can be complicated and risky.
+Certificate signing in Conjur is a capability intended to leverage the
+existing role and strengths of Conjur in an authorization architecture
+to promote the safe and effective use digital certificates.
 
-The policy for configuring a Conjur Certificate authority begins with
-a `webservice` entity with a well-known id of the form
-`conjur/<ca-service-id>/ca`. The CA is configured using well-known
-annotations. A minimal CA policy might look like:
+## Use Cases
+
+Conjur certificate authorities may be used to sign digital certificates
+addressing 4 use cases:
+
+1. **TLS Host Certificates**
+2. **TLS Client Certificates**
+3. **SSH Host Certificates**
+4. **SSH Client Certificates**
+
+## Getting Started with Conjur Certificate Authorities
+
+All Conjur Certificate Authorities (CA's) begin with a policy
+definition and a private key.
+
+### Create a Private Key
+In the alpha version, Conjur certificate authorities only support
+RSA private keys in PEM format. A valid private key may be created
+either using `openssl` or `ssh-keygen`:
+```sh
+openssl genrsa -aes256 -out private_key_file 4096 
+```
+```
+ssh-keygen -t rsa -m pem -f private_key_file
```

Comment:
I'd use the long form of the option flags here.

Reply:
I don't see long forms for these in the `ssh-keygen` docs. Do you have a resource for this, or were you assuming they would exist?